explainer.md

```diff
@@ -251,6 +255,10 @@ function drawScene(view) {
 }
 ```
 
+The `XRViewerPose` also contains a `transform`, which is an `XRRigidTransform` describing the position and orientation of the viewer as a whole relative to the `XRReferenceSpace` origin. This is primarily useful for rendering a visual representation of the viewer for spectator views or multi-user environments. Each `XRView` has a `transform` as well that can be used in lieu of the `viewMatrix` to position virtual cameras in the scene if the rendering library being used prefers.
```

Comment:
I think this is good point for further discussion, but I would like to perhaps do so in a different issue so that the core of this PR can land.

I would like to clarify something to make sure we're talking about the same thing: If we were to make your proposed change the `viewTransform` would still effectively represent the "camera" position/orientation, correct? Therefore if someone wanted to get a `viewMatrix` they'd have to invert the `viewTransform.matrix`. That feels weird to me, but I do expect that the assertion about most apps simply using the transform component directly is probably accurate, so maybe it's a moot point.

Reply:
> I would like to clarify something to make sure we're talking about the same thing: If we were to make your proposed change the `viewTransform` would still effectively represent the "camera" position/orientation, correct?

Yep!  The WebXR API then would not directly deliver an inverted left/right matrix for use as a view matrix - apps would instead construct their view matrices just as they would if no XR was involved at all: calculating them based on the app's camera position relative to the app's scene origin.  I have seen other new XR APIs increasingly take a similar approach to `viewTransform` data.

I'll create a new issue to discuss this.